### ISSUE TITLE
Error on install from Marketplace

### ISSUE THREAD

**Issue by stdword** [CorrectRoadH/logseq-gallery#2]
Message as Logseq error notification (UI):
```
[Main Exception]
ENOENT: no such file or directory, rename '/var/folders/yt/mkzzc_lj5wvbcvfgptg9vg7w0000gn/T/logseq-plugin-template-react-1.7.0.zip.pending' -> '/var/folders/yt/mkzzc_lj5wvbcvfgptg9vg7w0000gn/T/logseq-plugin-template-react-1.7.0.zip'
Error: ENOENT: no such file or directory, rename '/var/folders/yt/mkzzc_lj5wvbcvfgptg9vg7w0000gn/T/logseq-plugin-template-react-1.7.0.zip.pending' -> '/var/folders/yt/mkzzc_lj5wvbcvfgptg9vg7w0000gn/T/logseq-plugin-template-react-1.7.0.zip'
at Object.renameSync (node:fs:1042:3)
at PassThrough.<anonymous> (/Applications/Logseq.app/Contents/Resources/app/electron.js:14277:48)
at PassThrough.emit (node:events:514:28)
at endReadableNT (node:internal/streams/readable:1359:12)
at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
```

Plugin was installed, but didn't work:
<img width="600px" src="https://github.com/CorrectRoadH/logseq-gallery/assets/1984175/61c80ae7-2c86-4d3e-93ab-6c9a17ede65d"/>

**Comment by CorrectRoadH** [comment]
Thanks for your feedback. I try to reproduce it and fix it.🙏

**Comment by CorrectRoadH** [comment]
I found the cause of the error. The reason is the special query syntax such as "[[my book]]". I'll fix it tonight. 😉

**Comment by CorrectRoadH** [comment]
The Problem has been fixed in v1.7.1 😃
